python/bolt-mcp/src/bolt_mcp/server.py

```
@@ -176,18 +88,44 @@ def __init__(
+        clients are served: ``stateless=True`` disables sessions (multi-worker
+        safe, no server->client requests), ``json_response=True`` answers
+        legacy POSTs with plain JSON instead of SSE. 2026-07-28 clients are
+        always served statelessly regardless.
+
+        ``list_ttl_ms``/``list_cache_scope`` are the SEP-2549 caching hints
+        stamped on every list result (``0``/``"private"`` = no caching).
+        """
+        if list_cache_scope not in _CACHE_SCOPES:
+            raise ValueError(f"list_cache_scope must be 'public' or 'private', got {list_cache_scope!r}")
+        if list_ttl_ms < 0:
+            raise ValueError(f"list_ttl_ms must be >= 0, got {list_ttl_ms!r}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)